third_party/.gitignore

```diff
@@ -0,0 +1,8 @@
+/imgui/
+/stb/
+/KTX-Software/
+/Vulkan-Headers/
+/glm/
+/tinyobjloader/
+/cgltf/
+/tinyexr/
```

Comment:
Why the need to ignore these directories? Is anything getting picked up by git otherwise?

Reply:
Since these are no longer submodules and are optionally pulled in by cmake, then when they are downloaded they go to the above locations and then git picks them up and shows and you get 1000s of new files showing up. So it pollutes the git file changes which is why they are ignored